### PR TITLE
Add state test for create tx with maximum nonce

### PR DESCRIPTION
**Pull request by chfast**
The test check if the create transaction is reject if the origin's nonce
is maximum value (and would overflow if increased by 1).

### CODE REVIEW

File: src/GeneralStateTestsFiller/stCreateTest/CreateTransactionHighNonceFiller.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    currentDifficulty: 1
+    currentGasLimit: 1000000
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  expect:
+    - indexes:
+        data: !!int -1
+      network:
+        - London
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreateTest/CreateTransactionHighNonceFiller.yml]: why only on London? 
put `>=Merge`  too

File: src/GeneralStateTestsFiller/stCreateTest/CreateTransactionHighNonceFiller.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      expectException:
+        London: "TR_NonceHasMaxValue"
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreateTest/CreateTransactionHighNonceFiller.yml]: here it is also possible to type `>=Merge`

**Comment by winsvega** [comment]
when commiting state tests, run --filltests --fillchain command as well to generate the test version for hive 
thats why Travis check is red

**Comment by winsvega** [comment]
do you have access to push branches into tests repo? if so when you open a PR from branch inside of tests, I can finish your PRs if it requires filling or smth small easily.

**Comment by chfast** [comment]
> do you have access to push branches into tests repo? if so when you open a PR from branch inside of tests, I can finish your PRs if it requires filling or smth small easily.

I think I do, but hopefully I will improve over time.
